tests/cli/fvm_smoke_tests/fvm_basic.bats

```diff
@@ -809,3 +809,43 @@ setup_file() {
     rm -rf $FLUVIO_HOME_DIR
     assert_success
 }
+
+@test "Disallows switching versions when there are active proccesses" {
+    run bash -c '$FVM_BIN self install'
+    assert_success
+
+    # Sets `fvm` in the PATH using the "env" file included in the installation
+    source ~/.fvm/env
+
+    # Starts Fluvio Cluster
+    fluvio cluster delete --local || true
+
+    # Installs Fluvio Stable
+    run bash -c 'fvm install'
+    assert_success
+
+    # Starts Fluvio Cluster
+    run bash -c 'fluvio cluster start --local'
+    assert_success
+
+    # Attempts to switch version
+    run bash -c 'fvm install latest'
+    assert_line --index 6 "Error: Cannot switch versions while \`fluvio\` or \`fluvio-run\` are running"
+    assert_failure
+
+    # Deletes Fluvio Cluster
+    run bash -c 'fluvio cluster delete'
```

Comment:
I'd also make this '--local'

Reply:
The `delete` no longer accepts such parameter. It reads everything it needs from the profile. If it tries to delete the k8s cluster when you expect it deletes local, it means somewhere was an attempt to start k8s cluster and the current profile is not the one you expect